src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -1978,66 +1978,80 @@ private void checkpoint() {
             // Nothing to persist OR we've already persisted it along the way.
             return;
         }
+        LOGGER.log(Level.INFO, "Attempting to save a checkpoint of {0} before shutdown", this);
```

Comment:
This message and the one at the end of `CpsFlowExecution.checkpoint` are now the only messages that are logged at INFO level if everything is successful. Any errors are logged at WARNING level.